deployments/bridges/rialto-millau/dashboard/grafana/provisioning/dashboards/relay-rialto-to-millau-headers-dashboard.json

```diff
@@ -0,0 +1,694 @@
+{
```

Comment:
Do we really need two separate dashboards? I think it would be more ergonomic to present both information on a single dashboard.

Reply:
I'm unsure if that'll be the case. It might be more confusing to have everything in one. I can play around with it the idea though